s3backuprestore/restore.py

```diff
@@ -137,14 +139,18 @@ def run(self):
             # If objects storage class equals GLACIER put it into
             # _glacier_queue to process it later.
             if storage_class and 'GLACIER' in storage_class:
-                logger.warning("{} in Glacier. Waiting {}s."
-                               .format(key, waiter))
+                logger.warning("Checking for ongoing-request. "
```

Comment:
Is this really reason for a warning? Something that happens nearly every time. Maybe use debug instead?

Reply:
Done